pkg/handlers/archive.go

```diff
@@ -152,9 +153,13 @@ func (h *archiveHandler) openArchive(
 		}
 		defer rdr.Close()
 
-		return h.openArchive(ctx, depth+1, rdr, dataOrErrChan)
+		// Note: We're limited in our ability to add file names to the archiveEntryPath here, as the decompressor doesn't have access to a fileName value. Instead, we're adding a generic string to indicate that the file is decompressed. This could be improved.
```

Comment:
@mcastorina any thoughts on how to make this a little cleaner? I imagine an edge case of a zip file with 10 `.gz` files nested and then we just say `decompressed .gz file` and it's super unclear.